en/SUMMARY.md

```diff
@@ -15,8 +15,10 @@
 * [Get Support](What_next/getSupport.md)
 * [Developer Checklist](What_next/devChecklist.md)
 * Appendices
-   * [Appendix A: Learning Resources](Appendices/appendixA.md)
-   * [Appendix B: OpenMRS Glossary](Appendices/appendixB.md)
-   * [Appendix C: Troubleshooting](Appendices/appendixC.md)
+  * [Appendix A: Learning Resources](Appendices/appendixA.md)
+  * [Appendix B: OpenMRS Glossary](Appendices/appendixB.md)
+  * [Appendix C: Troubleshooting](Appendices/appendixC.md)
+    * [Software and Development](Appendices/appendixC/software-and-development-troubleshooting.md)
```

Comment:
Should this be here?

Reply:
Yes, I intentionally split the Troubleshooting appendix into two sub-directories. I left more general content in the main page of Appendix C.